build.sbt

```diff
@@ -2,16 +2,20 @@ organization in ThisBuild := "com.stripe"
 
 scalaVersion in ThisBuild := "2.11.5"
 
-crossScalaVersions in ThisBuild := Seq("2.10.4", "2.11.5")
+crossScalaVersions in ThisBuild := Seq("2.10.4", "2.11.5", "2.12.1")
 
 scalacOptions in ThisBuild ++= Seq(
-  "-Yinline-warnings",
   "-deprecation",
   "-feature",
   "-unchecked",
   "-optimize"
 )
 
+scalacOptions in ThisBuild ++= (scalaBinaryVersion.value match {
+  case "2.12" => Seq.empty
```

Comment:
Scala 2.12 has a rewritten optimizer that is much more robust and checks whether method can be inlined at the definition site so you don't get inliner warnings anymore.